src/node_report.h

```diff
@@ -49,6 +35,21 @@ void WriteReport(const v8::FunctionCallbackInfo<v8::Value>& info);
 void GetReport(const v8::FunctionCallbackInfo<v8::Value>& info);
 
 }  // namespace report
+
+// Function declarations - functions in src/node_report.cc
```

Comment:
aren't these duplicates of the declarations in node.h (which should be the source of truth, not these)?

Reply:
Thanks for pointing this out. Removed.